### PR TITLE
Initial changes for the project

### PR DESCRIPTION
**Pull request by kvasdopil**

### CODE REVIEW

File: src/banner.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import QueryString from 'query-string';
+
+import AppBanner from './components/AppBanner';
+
+function init(query) {
```
Comment (GU5TAF) [inline src/banner.js]: `query` argument should probably be called `props`, that's how you're using it.

File: src/components/BannerWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```
Comment (GU5TAF) [inline src/components/BannerWrapper.jsx]: `import React, { Component } from 'react';` is tidier.

File: src/components/BannerWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+export default Component => class BannerWrapper extends React.Component {
```
Comment (GU5TAF) [inline src/components/BannerWrapper.jsx]: If using the above import style you can just:
`... extends Component {`

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+//
+// ...
+// <div>
+//  <AppBanner google="com.presencekit.eyerim" apple="id1184932325" position="bottom-left" />
+// </div>
+//
+
+import AppBanner from './components/AppBanner';
+
+export default AppBanner;
```
Comment (GU5TAF) [inline src/index.js]: importing and subsequently exporting can be done in one line:
`export { default } from './components/AppBanner';`

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    loaders: [{
+      test: /\.jsx?$/,
+      exclude: /(node_modules)/,
+      loader: 'babel-loader',
+    }, {
+      test: /\.scss$/,
+      loader: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!sass-loader',
+    }, {
+      test: /\.svg$/,
+      loader: 'url-loader',
```
Comment (GU5TAF) [inline webpack.config.js]: > The url-loader works like the file-loader, but can return a data URL if the file is smaller than a byte limit.
> The limit can be specified with a query parameter (defaults to no limit).
> If the file is greater than the limit (in bytes) the file-loader is used and all query parameters are passed to it.

You haven't specified a limit, so everything is being inlined. Is this the desired behaviour?

Comment (kvasdopil) [inline webpack.config.js]: Yeah, the idea was to get rid of as much external files as possible.

Comment (GU5TAF) [inline webpack.config.js]: 👍

File: src/components/BannerWrapper.jsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+    this.state = {
+      status: 'shown',
+    };
+
+    this.timer = undefined;
+  }
+
+  onSuccess() {
+    const { timeout = 3000 } = this.props;
```
Comment (GU5TAF) [inline src/components/BannerWrapper.jsx]: Use `defaultProps` instead of setting a fallback here.

File: src/components/BannerWrapper.jsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    this.timer = setTimeout(() => this.dismiss(), timeout);
+  }
+
+
+  onFailure() {
+    this.setState({ status: 'error' });
+  }
+
+  dismiss() {
+    const { onDismiss = () => {} } = this.props;
```
Comment (GU5TAF) [inline src/components/BannerWrapper.jsx]: Use `defaultProps` instead of setting a fallback here.

Comment (kvasdopil) [inline src/components/BannerWrapper.jsx]: done

File: src/components/BannerWrapper.jsx (hunk 6 above)
Comment (kvasdopil) [inline src/components/BannerWrapper.jsx]: done

**Pull request merged**